### PR TITLE
tooling(git): fix up `.gitkeep`

### PR DESCRIPTION
**Pull request by pvinis**
The type of this PR is: **Enhancement**

This PR resolves []

### Description

I noticed that a few of these `.gitkeep` files were in our repo, and they were not needed. One is needed, in `Emission`, but we did not have it in our gitignore, so basically it was accidentally kept.

Now we have the files cleaned up, and we actually make sure to keep them.

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- Use `.gitkeep` files correctly - pavlos

</details>

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Dev changes (Use `.gitkeep` files correctly - pavlos)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against f075ca9bc4e3be8ab3f660036bb35215977061d5
</p>

File: .gitignore [hunk 1]
```diff
@@ -140,3 +140,7 @@ android/app/google-services.json
 GoogleService-Info.plist
 
 storybook.json
+
+
+# Keep these, so we can keep the folders these appear in
+!.gitkeep
```
Comment (brainbicycle) [inline .gitignore:146]: isn't this default behavior?

**Review by brainbicycle — APPROVED** [review]

**Comment by brainbicycle** [comment]
One question, non-blocking, thanks for cleaning this up! Also TIL this isn't a git feature 😅 : https://stackoverflow.com/questions/7229885/what-are-the-differences-between-gitignore-and-gitkeep

File: .gitignore (hunk 1 above)
Comment (pvinis) [reply .gitignore:146]: No, this it not a default git behavior. It's just a generally accepted hack/convention :D

**Pull request merged**